packages/@ngtools/webpack/src/loader.ts

```diff
@@ -201,6 +202,15 @@ export function ngcLoader(source: string) {
       })
       .then(() => {
         if (plugin.typeCheck) {
+          // Check all diagnostics from this and reverse dependencies also.
+          if (!plugin.firstRun) {
+            this._module.reasons
+              .filter(reason => reason.module && reason.module instanceof NormalModule)
+              .forEach(reason => plugin.diagnose(reason.module.resource));
```

Comment:
Yeah I was doing that, and forgot to put it back, my bad. Thanks for catching it!